datetime_formatter/models/res_lang.py

```diff
@@ -105,7 +105,7 @@ def datetime_formatter(self, value, lang=None, template=MODE_DATETIME,
         # Convert str to datetime objects
         if isinstance(value, str):
             try:
-                value = fields.Datetime.from_string(value)
+                value = fields.Datetime.to_datetime(value)
             except ValueError:
                 # Probably failed due to value being only time
                 value = datetime.strptime(value, DEFAULT_SERVER_TIME_FORMAT)
```

Comment:
I'm not sure this is still needed.

Reply:
@ernestotejeda can you check this?

Reply:
Well, I checked it now and it's true Odoo 11 returns date/datetime fields as string in Qweb, but Odoo 12 returns date/datetime fields as the corresponding python date/datetime object. Anyway, we could keep it this way as safeguard, as a tool in case someone gets the value of a field in date/datetime directly from DB using a select query and wants to format that field according to a certain language